scripts/mk_repo_file.py

```diff
@@ -232,6 +248,7 @@ def main() -> None:
     parser.add_argument('--include-prefix', action="append", default=[],
                         help="only include paths if they start with the specified prefix")
     parser.add_argument('--exclude-path', action="append", default=[], help="paths to exclude from fetching")
+    parser.add_argument('--jobs', '-j', default=16, type=int, help="number of concurrent jobs")
```

Comment:
Can this default to a detected number of cores?

Reply:
This should definitely be auto-detected.

Reply:
Perhaps something needs to be done in line 84 too